server/db/models/wiki.js

```diff
@@ -0,0 +1,10 @@
+const Sequelize = require('sequelize');
+const db = require('../builder');
+
+const Wiki = db.define('wiki', {
```

Comment:
@AnnaC91 @ErikaNewland same as NYT model, we should define these internally: 

1) gets us all on the same page using the same jargon
2) better aligns with naming conventions ?? 

Or sorry I'm confused how the WIKI is different than the Experiences model- even though the Wiki can start off different experiences the content on the wiki doesn't change- and the links to different headlines can still be different based on how we query it... 

It seems to me that "Experience" can be the Route, like where one articles takes another and how that isunique, but the wiki article in itself will never change? the route or the "experience" will change so i think we need to reconsider the relationship there

Reply:
@AnnaC91 `Synopsis` instead of wiki with a type column sequelize.string